src/main/resources/tag_profile.yaml

```diff
@@ -0,0 +1,20 @@
+name: Tag Profile
+tagMappings:
+  - value: 69
+    valueType: engId
+    tags: [ RHEL, RHEL for x86 ]
+  - value: x86_64
+    valueType: arch
+    tags: [ RHEL for x86 ]
+tagMetrics:
+  - tag: Openshift-metrics
+    metricId: Cores
+    Uom: Cores
```

Comment:
Please change to 'uom' to be consistent.

A value of 'Cores' will not work here. Unless I'm mistaken, it has to match the case of the Enum variable.
```CORES```

I think this is the cause of the mass test failures I'm seeing.